### ISSUE TITLE
Realtime alerts should group whenever possible

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#3102]
I signed up for real time alerts for all oral arguments, and today I got three emails at almost exactly the same time because we scraped three items from the DC circuit all at once. Instead of doing that, we should send one alert with all three. 

This will become more important as we move to RECAP, because we'll be getting dozens or even hundreds of docket entries at a time, and we should be sure we only send one email when that happens. 

One solution is to delay RT alerts by a few minutes, but that's pretty unsatisfying (this is how Stackoverflow does it, but still). If we can, we should try to be smarter about how we send. Some of our users insist that ever second counts.

**Comment by albertisfu** [comment]
This is done for RECAP in #4200 I'll use this issue to apply grouping also for Oral arguments RT alerts.

**Comment by mlissner** [comment]
To be clear, I think this applies across all RT alerts, no?

**Comment by albertisfu** [comment]
Yeah, that's correct. For now, we need to apply it to Oral Argument RT alerts, which is the other document type supported by the Percolator.
Opinions Alerts will continue using the current `cl_send_alerts` approach, with the difference of using Elasticsearch (instead of Solr) to query alerts. There grouping is already applied for RT alerts. We'll need to consider grouping for Opinions Alerts once we move them to the Percolator/Sweep index approach which I don't think we have an issue for that yet. Should we create one?

**Comment by mlissner** [comment]
Yep, let's get on it. :)

**Comment by mlissner** [comment]
What are we holding this open for?

**Comment by albertisfu** [comment]
Yes, this is still open since the review of https://github.com/freelawproject/courtlistener/pull/4251 is pending.

**Comment by albertisfu** [comment]
Closing this one since https://github.com/freelawproject/courtlistener/pull/4251 has been merged. Now, both RECAP and OA search alerts are grouped. For Opinions this is within the scope of: https://github.com/freelawproject/courtlistener/issues/4247
